### DISCUSSION TITLE
Wildcards in Pools, Databases & Users

### DISCUSSION THREAD

**Discussion by sherifkayad** [postgresml/pgcat#368] [General]
Hello,

We currently have a setup where:
- We have multiple databases (~50 DBs => one DB for each service)
- We have multiple users (tons of human users + ~50 app users)

I was wondering if it's possible to whitelist all databases and all users without having to explicitly define these as `pools.DB_NAME` & `pools.DB_NAME.X`, where X would be:
- The pool users
- The pool shards

What I am looking for is something like `ANY` allowing any user (and leaving the auth to the underlying PostgreSQL DB) for any Database (leaving the database itself to the underlying PostgreSQL DB).

Thanks a lot.

**Comment by levkk** [comment]
I think you may be looking for this: https://github.com/levkk/pgcat/issues/362
